sechub-schedule/src/main/java/com/daimler/sechub/domain/schedule/strategy/SchedulerStrategy.java

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.domain.schedule.strategy;
+
+import java.util.UUID;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Please remove @Component from interfaces, the annotation should only be used inside implementations but not interfaces.